### PR TITLE
Display craftname in osd preview & fix 'E' character in Betaflight font

### PR DESCRIPTION
**Pull request by basdelfos**
- Display craftname in OSD preview instead of CRAFT_NAME (if not empty).
- Fix 'E' character in Betaflight font.

### CODE REVIEW

**Comment by mikeller** [comment]
Fixes #551.

File: tabs/osd.js [hunk 1]
```diff
@@ -383,7 +390,10 @@ OSD.constants = {
       desc: 'osdDescElementCraftName',
       default_position: -77,
       positionable: true,
-      preview: 'CRAFT_NAME'
+      preview: function(osd_data) {
+        return OSD.generateCraftName(osd_data, 1);
+      }
+      //preview: 'c'
```
Comment (mikeller) [inline tabs/osd.js]: What is this comment for?

Comment (basdelfos) [inline tabs/osd.js]: Removed in next commit

**Comment by DanNixon** [comment]
(@mikeller If you want that comment to automatically close the issue when merged then it has to either be in the PR description or a commit message)

**Pull request merged**